src/libraries/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests.csproj

```diff
@@ -2,9 +2,10 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
-    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-unix;$(NetCoreAppCurrent)-browser</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-unix;$(NetCoreAppCurrent)-browser;$(NetCoreAppCurrent)-osx</TargetFrameworks>
     <IgnoreForCI Condition="'$(TargetOS)' == 'browser'">true</IgnoreForCI>
     <EnableLibraryImportGenerator>true</EnableLibraryImportGenerator>
+    <StringResourcesPath>$(MSBuildProjectDirectory)\..\src\Resources\Strings.resx</StringResourcesPath>
```

Comment:
We have two options:

1. Include the two keys in test resources:
    ```xml
	  <data name="CantGetAllPids" xml:space="preserve">
	    <value>Could not get all running Process IDs.</value>
	  </data>
	  <data name="RUsageFailure" xml:space="preserve">
	    <value>Failed to set or retrieve rusage information. See the error code for OS-specific error information.</value>
	  </data>
    ```
    in:
    https://github.com/dotnet/runtime/blob/b4a3c9e6fbe09d2f52465e2189d57ef5c04b94c3/src/libraries/System.Diagnostics.Process/tests/Resources/Strings.resx#L1
    and delete this hard-coded line.

2. Include test resources alongside source ones -- separated by semicolon `;`:
    ```xml
		 <StringResourcesPath>$(MSBuildProjectDirectory)\Resources\Strings.resx;$(MSBuildProjectDirectory)\..\src\Resources\Strings.resx</StringResourcesPath>
    ```

    multiple resources seem to be supported.

Option 1 is probably much cleaner that avoids mixing stuff, but I'll defer to others. cc @ViktorHofer

Reply:
Agreed. Option 1 is cleaner while it adds some duplication.